index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <link rel="stylesheet" href="style.css">
+    <title>Luk, A man</title>
+</head>
+<body>
+    <header>
+        <div class="header"> 
+            <div class="headertext">Welcome</div>
+            <div>
+            <label for="trigger">
+            <input type="checkbox" class="trigger" id="trigger"/>
+            <div class="hamburger">
+                <span></span>
+                <span></span>
+                <span></span>
+            </div>
```

Comment:
- [ ] Kindly make sure to wrap your hamburger icon into the `nav` tag in order to correctly format your HTML semantically. :wink: